src/test/java/unit/BqTableReferenced.java

```diff
@@ -381,6 +398,30 @@ void updateIndividualProperties() throws IOException {
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqTable.class, "resource", "describe", "--name=" + newName);
     assertEquals(newDescription, describeDataTable.description);
+
+    updateDataTable =
+        TestCommand.runAndParseCommandExpectSuccess(
+            UFBqTable.class,
+            "resource",
+            "update",
+            "bq-table",
+            "--name=" + newName,
+            "--new-table-id=" + externalDataTableName2);
+    assertEquals(externalDataTableName2, updateDataTable.dataTableId);
+    assertEquals(externalDataset.getDatasetId(), updateDataTable.datasetId);
+    assertEquals(externalDataset.getProjectId(), updateDataTable.projectId);
+
+    updateDataTable =
+        TestCommand.runAndParseCommandExpectSuccess(
+            UFBqTable.class,
+            "resource",
+            "update",
+            "bq-table",
+            "--name=" + newName,
+            "--new-dataset-id=" + externalDataset2.getDatasetId());
+    assertEquals(externalDataTableName2, updateDataTable.dataTableId);
+    assertEquals(externalDataset2.getDatasetId(), updateDataTable.datasetId);
+    assertEquals(externalDataset2.getProjectId(), updateDataTable.projectId);
```

Comment:
same question as above -- why did the project id here also change, shouldn't it just be the dataset id?

Reply:
same as above.